flask_mailgun.py

```diff
@@ -73,18 +97,72 @@ def init_app(self, app):
         this, it is an email address where messages will be sent.
         Optionally, you can configure the sender address as well
         with the MAILGUN_LOGGING_SENDER option.
+
+        There are also several template strings that can be optionally
+        configured. If not configured, they fall back to default
+        values. These options are:
+          - MAILGUN_API_URL_TEMPLATE
+          - MAILGUN_DEBUG_TEMPLATE
+          - MAILGUN_LOGGING_TEMPLATE
+          - MAILGUN_ERROR_SUBJECT_WITHEXCINFO_TEMPLATE
+          - MAILGUN_ERROR_SUBJECT_WITHOUTEXCINFO_TEMPLATE
+          - MAILGUN_ERROR_TEMPLATE
+          - MAILGUN_API_ERROR_TEMPLATE
         """
         app.extensions['mailgun'] = self
         self.debug = app.debug
+
+        self.domain = app.config['MAILGUN_DOMAIN']
+        self.key = app.config['MAILGUN_KEY']
+
+        self.api_url_template = app.config.get(
```

Comment:
Ok, I have changed this as you requested. I realise that the one-line version is more Pythonic, but the "is None" version lets people override these to be an empty string if they want (although not sure why anyone would want that).